viewing_party/party.py

```diff
@@ -1,23 +1,185 @@
 # ------------- WAVE 1 --------------------
-
+#---Wave_1_function_1---
 def create_movie(title, genre, rating):
-    pass
 
-# -----------------------------------------
+    movie_dict = {}
+
+    if not title:
+        return None
+    elif not genre:
+        return None
+    elif not rating:
+        return None
+    else: 

```

Comment:
Good `if/elif/else` conditionals! The line `return None` is repeated a few times. It might be nice to refactor and combine some of these conditional cases. DRY!